Source/ACE.Server/Managers/LandblockManager.cs

```diff
@@ -59,24 +60,28 @@ public static List<Landblock> GetActiveLandblocks()
         }
 
         /// <summary>
-        /// This should only be used for debugging/development purposes.
+        /// Loads the specified list of landblocks and optionally their adjacents.
         /// </summary>
-        public static void ForceLoadLandBlock(LandblockId blockid)
+        public static void ForceLoadLandBlock(LandblockId blockid, bool propagate = false)
```

Comment:
The arguments for this function should be the same as GetLandblock:
LandblockId landblockId, bool propagate, bool permaload = false

Reply:
I'm not sure that I understand why the arguments need to be the same.  The function originally statically defined bool propagate as false.  For landblock preloading, it makes sense to have it be true, hence the option argument for propagate with a default of false so as to not break anything already using ForceLoadLandBlock().  When forcing a landblock to load, it doesn't really make sense to allow the same landblock to then unload itself five minutes later, which is why I didn't see any issue with making that a static assignment for the internal function call.

Reply:
They should be the same to give the caller the power to chose.

In the current implementation, every landblock loaded via Force is set to permaload. That's not an assumption we can make every time.

I added ForceLoadLandblock, and it's only used in one place, by a developer load test command.

It doesn't hurt to have the signatures the same and let the caller decide if the landblock should propegate and permaload.